### ISSUE TITLE
\multiline not working right

### ISSUE THREAD

**Issue by OlgaRedozubova** [mathjax/MathJax-src#269]
We aren't rendering \multiline's properly. For example take this:
```
\begin{multline*}
p(x) = 3x^6 + 14x^5y + 590x^4y^2 + 19x^3y^3\\ 
- 12x^2y^4 - 12xy^5 + 2y^6 - a^3b^3
\end{multline*}
```
![screenshot-newtab-2019 05 30-08-42-28](https://user-images.githubusercontent.com/32493105/58611243-200f2400-82b7-11e9-9072-101af0da08dd.png)

After the `\\` is missing part of the equation.

**Comment by OlgaRedozubova** [comment]
Perhaps in this case the width of the container is incorrectly calculated for "svg".

![screenshot-newtab-2019 05 30-08-52-19](https://user-images.githubusercontent.com/32493105/58611689-a7a96280-82b8-11e9-82da-b3918e70f114.png)

**Comment by dpvc** [comment]
Thanks for the report.  The issue is more with the example file than with MathJax itself.  The particular demo that you are using is one of the ones that illustrate the use of MathJax's non-document-based conversion functions; this one takes a TeX string and returns a serialized SVG string (which it then inserts into the document for display).  

Because the conversion happens outside of a browser DOM, MathJax needs to be told the metric information like the em-size, ex-size, and container width.  The demo is not very sophisticated, and just uses the defaults for these, which are a 16px em, an 8px ex, and a very large container width (to indicate "infinite").

For HTML output, this is not a problem, but for SVG output, a table that is set to have 100% width (like the one in the `multline` environment) ends up being excessively large, as you have pointed out above.  I will need to modify the demo in order to set the width to an appropriate value (and perhaps change the defaults as well).

If you use the [in-page typesetting example](https://mathjax.github.io/mj3-demos/input-tex_mml2svg.html) instead, you will see that the layout is correct when the metrics are properly determined.

**Comment by dpvc** [comment]
This is working in the latest versions of the demos.
